src/pages/commits/ActiveCommits/Grid.tsx

```diff
@@ -8,8 +8,8 @@ export const Grid: React.FC<{
   numDashedLine: number;
 }> = ({ numDashedLine }) => (
   <ColumnContainer>
-    {Array.from(Array(numDashedLine).keys()).map((number) => (
-      <DashedLine key={number} />
+    {[...Array(numDashedLine)].map((number) => (
+      <DashedLine key={`${number}_grid_line`} />
```

Comment:
This is returning `undefined_grid_line` as the key

Reply:
nice catch just updated it. I still don't like this syntax but thats i could't find an alternative i liked better.